eggplant/market/models/inventory.py

```diff
@@ -40,12 +47,15 @@ class Meta:
 class ProductCategory(models.Model):
 
     title = models.CharField(max_length=70)
-    description = models.TextField()
+    description = models.TextField(null=True, blank=True)
```

Comment:
Now you can add a product category without a description -- but is that handled when displaying templates? We could also remove the whole description thing until it serves a well-known purpose enough to write the field's `help_text` ?